src/api/app/controllers/webui/request_controller.rb

```diff
@@ -16,6 +16,7 @@ class Webui::RequestController < Webui::WebuiController
                              if: -> { Flipper.enabled?(:request_show_redesign, User.session) }
   before_action :set_influxdb_data_request_actions, only: %i[show build_results rpm_lint changes mentioned_issues],
                                                     if: -> { Flipper.enabled?(:request_show_redesign, User.session) }
+  before_action :set_diff_to_superseded_id, only: %i[show request_action request_action_changes build_results rpm_lint changes mentioned_issues]
```

Comment:
@ncounter one more thing, couldn't this also be done in `:set_superseded_request`? IMO its not worth to create another callback and method just to set this variable.